components/taghints.py

```diff
@@ -244,12 +244,13 @@
     },
     "compat": {
         "message": (
-            "{query} You seem to be using a version &gt;=20.0 of <code>python-telegram-bot</code> "
-            " but your code is written for version &lt;=13.15. Please either upgrade your code to "
-            " v20 by reading the release notes and the transition guide linked below or install "
-            "a version of PTB that is compatible with your code base.\n"
-            "Please note that we only provide support for the latest stable version and that the "
-            "library has undergone significant changes in v20."
+            "{query} You seem to be using the new version (&gt;=20.0) of"
+            "<code>python-telegram-bot</code> but your code is written for an older and "
+            "deprecated version (&lt;=13.15).\nPlease update your code to the new v20 by reading"
+            " the release notes and the transition guide linked below.\nYou can also install a "
+            "version of PTB that is compatible with your code base, but please note that the "
+            "library has undergone significant changes in v20 and the older version is not "
+            "supported anymore and may be broken."
```

Comment:
Maybe "broken" can be made a bit more precise?

```suggestion
            "supported anymore. It may contain bugs that that will not be fixed by the PTB team and it also doesn't support functionality of Bot API X.Y+."
```

Reply:
I committed a more explanatory and future-ready phrase without introducing two new variables